streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -315,6 +316,9 @@ private TopologyTestDriver(final InternalTopologyBuilder builder,
         configCopy.putIfAbsent(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "dummy-bootstrap-host:0");
         // provide randomized dummy app-id if it's not specified
         configCopy.putIfAbsent(StreamsConfig.APPLICATION_ID_CONFIG,  "dummy-topology-test-driver-app-id-" + ThreadLocalRandom.current().nextInt());
+        // provide default serde since we change default serde to be null in KIP-741
+        configCopy.putIfAbsent(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.ByteArraySerde.class);
+        configCopy.putIfAbsent(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.ByteArraySerde.class);
```

Comment:
Since the `TopologyTestDriver` should test a topology and somehow the default serdes are part of the topology under test, the `TopologyTestDriver` should fail, if no default serdes are provided by the users but needed. Hence, I think, it is not a good idea to silently add the default serdes here. 
That means, you have to provide deserializers and serializers in some tests in `TopologyTestDriverTest`. For that look for methods called `setup*()` (e.g. `setupSourceSinkTopology()`) and add the deserializers and serializers there. In some tests, you need to add them directly in the test.

Reply:
Thank you @cadonna , I also find a test `ProcessorNodeTest#testTopologyLevelConfigException`  should fail on default config, I moved `properties` to every releated test cases.